image-types/rhel8/README.md

```diff
@@ -0,0 +1,21 @@
+# Red Hat Enterprise Linux 8
+
+Owner: *None*
+
+*osbuild-composer* can create [Red Hat Enterprise Linux 8][rhel] images.
+
+## Architectures
+
+RHEL supports `x86_64`, `aarch64`, `ppc64le`, and `s390x`.
+
+## File System
+
+The default file system is `xfs`.
```

Comment:
Implicit: only one partition, right?

Reply:
I had that on both the specific documents (ec2 and openstack), but yeah, probably makes sense to add here as well as a default.